CONFIGDOC.md

```diff
@@ -97,6 +108,201 @@ You can define several axes in the **Configuration Matrix** section. For example
+                               codeCoverageCobertura: 'code-coverage/coverage.xml')
+            }       
+        }                
+    } 
+}
+``` 
+
+Define a Scripted Pipeline to run your tests and generate artifacts.
+
+```groovy
+node {
+    runMATLABTests(testResultsJUnit: 'test-results/results.xml',
+                   codeCoverageCobertura: 'code-coverage/coverage.xml') 
+}
+``` 
+
+
+ 
+ This table lists the name-value pair arguments that you can specify with the `runMATLABTests` step.
+
+| Name               	| Value                                                                                                                	|
+|-------------------------	|----------------------------------------------------------------------------------------------------------------------------	|
+| testResultsPDF          	| Path to write test results report in PDF format (currently not supported on Mac platforms).<br/>**Example:** `'test-results/results.pdf'`                  	|
+| testResultsTAP          	| Path to write test results report in TAP format.<br/>**Example:** `'test-results/results.tap'`                  	|
+| testResultsJUnit        	| Path to write test results report in JUnit XML format.<br/>**Example:** `'test-results/results.xml'`            	|
+| testResultsSimulinkTest 	| Path to export Simulink Test&trade; Manager results in MLDATX format (requires Simulink Test license and is supported in MATLAB R2019a or later).<br/>**Example:** `'test-results/results.mldatx'` 	|
+| codeCoverageCobertura   	| Path to write code coverage report in Cobertura XML format.<br/>**Example:** `'code-coverage/coverage.xml'`     	|
+| modelCoverageCobertura  	| Path to write model coverage report in Cobertura XML format (requires Simulink Coverage™ license and is supported in MATLAB R2018b or later).<br/>**Example:** `'model-coverage/coverage.xml'`   	|
+
+
+## Use MATLAB for Matrix Build
+Similar to multi-configuration projects, you can use the plugin to perform [matrix](https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-matrix) builds in Pipeline projects. For example, you can define a Declarative Pipeline to run your test suite on different platforms or against different versions of MATLAB. (Matrix build is not supproted in Scripted Pipelines.)
+
+Your Pipeline must have a `matrix` section to define the possible name-value combinations that should run in parallel. This example shows how to define a Pipeline to run your MATLAB code and generate test artifacts using MATLAB R2018b, R2019a, and R2020a.
+
+```groovy
+pipeline {
+    agent any
+    stages {
+        stage('BuildAndTest') {
+            matrix {
+                agent any
+                environment {
+                    PATH = "C:\\Program Files\\MATLAB\\${VERSION}\\bin;${PATH}"   // Windows agent
+                }
+                axes {
+                    axis {
+                        name 'VERSION'

```

Comment:
Maybe clearer if this variable was name `MATLAB_RELEASE` or `MATLAB_VERSION`.

Reply:
Done.